### PR TITLE
Fix user list filter bug (#897)

### PR DESCRIPTION
**Pull request by a2batic**
Tendrl-bug-id: #897 
Signed-off-by: a2batic <murarkakanika@gmail.com>

### CODE REVIEW

**Review by gnehapk — DISMISSED** [review]

File: src/modules/tasks/global-task-detail/global-task-detail.js [hunk 1]
```diff
@@ -1,120 +1,116 @@
...
+            templateUrl: "/modules/tasks/global-task-detail/global-task-detail.html",
+            bindings: {},
+            controller: globalTaskDetailController,
+            controllerAs: "glbTaskDetailCntrl"
+        });
 
-        /*@ngInject*/
-        function globalTaskDetailController($rootScope, $scope, $interval, $state, $stateParams, taskStore, config, utils, clusterStore) {
+    /*@ngInject*/
+    function globalTaskDetailController($rootScope, $scope, $interval, $state, $stateParams, taskStore, config, utils, clusterStore) {
```
Comment (cloudbehl) [inline src/modules/tasks/global-task-detail/global-task-detail.js]: I don't see the use of "utils" service in this controller, please remove the extra dependencies if there is any.

File: src/modules/users/user-list/users.js [hunk 2]
```diff
@@ -16,7 +16,6 @@
     function userController($scope, $rootScope, $state, $uibModal, $interval, utils, config, userStore, AuthManager, Notifications) {
```
Comment (cloudbehl) [inline src/modules/users/user-list/users.js]: I don't see the use of "utils" service in this controller, please remove the extra dependencies if there is any.

**Review by cloudbehl — CHANGES_REQUESTED** [review]

File: src/modules/tasks/global-task-detail/global-task-detail.js (hunk 1 above)
Comment (a2batic) [reply src/modules/tasks/global-task-detail/global-task-detail.js]: Acknowledge

File: src/modules/users/user-list/users.js (hunk 2 above)
Comment (a2batic) [reply src/modules/users/user-list/users.js]: Acknowledge

**Review by gnehapk — DISMISSED** [review]

**Review by gnehapk — APPROVED** [review]

**Review by cloudbehl — APPROVED** [review]
LGTM

**Review by cybernth — APPROVED** [review]

**Pull request merged**